crates/oxc_linter/src/utils/react.rs

```diff
@@ -12,16 +12,13 @@ use oxc_semantic::AstNode;
 
 use crate::{LintContext, OxlintSettings};
 
-pub fn is_create_element_call(call_expr: &CallExpression) -> bool {
+pub fn is_create_element_call(call_expr: &CallExpression, ctx: &LintContext) -> bool {
     match &call_expr.callee {
-        Expression::StaticMemberExpression(member_expr) => {
-            member_expr.property.name == "createElement"
-        }
-        Expression::ComputedMemberExpression(member_expr) => {
-            member_expr.static_property_name().is_some_and(|name| name == "createElement")
-        }
         Expression::Identifier(ident) => ident.name == "createElement",
-        _ => false,
+        expr => expr.is_specific_member_access(
+            ctx.settings().react.pragma.as_ref().map_or("React", |pragma| pragma.as_str()),
+            "createElement",
+        ),
```

Comment:
There are also some false positive:

```js
const createElement = document.createElement
createElement('iframe')

const React = myReact
React.createElement('iframe')
```


Can we port  [`isDestructuredFromPragmaImport`](https://github.com/jsx-eslint/eslint-plugin-react/blob/d1556a3d2f95f56cfb6e7c0a341af16742f64033/lib/util/isCreateElement.js#L28-L29) here? we can ignore the `require` part for this issue.

Reply:
Or don't add `ctx.settings().react.pragma` for this issue:

https://github.com/oxc-project/oxc/blob/6f2253886ddb461063acc45e7647c2805adc4d65/crates/oxc_linter/src/utils/react.rs#L147